neofetch

```diff
@@ -6142,6 +6142,34 @@ ${c1}                    y:
 EOF
         ;;
 
+        "aerOS"*)
+            set_colors 6 6 7 1
+            read -rd '' ascii_data <<'EOF'
+${c1}                    y:
```

Comment:
Is the `y:` here intentional? It would look like a nice circle otherwise 💭 

![image](https://github.com/dylanaraps/neofetch/assets/22280294/6ab6c628-5a17-4b57-879c-ee10e348d6be)

Reply:
@hykilpikonna remove the y: its just a circle, also change color to white.

btw these changes are made by @XanuduBuster and i think he set it to blue instead of white. but the y: is a misake.